### PR TITLE
Demo script coupling

### PR DESCRIPTION
**Pull request by robinborst95**
Added some new features that are useful for the visualization.
- the table gui is now fullscreen
- the record now keeps track of the order of keys.
- in preview table you can now choose the correct table.
- the exporter adds chunk and code columns

### CODE REVIEW

**Comment by paulbakker65** [comment]
Looks great, useful additions and no problems to be found in the maven site

**Comment by grootstyr** [comment]
SingleConditionPattern gets printed to system.out in SingleOccurrenceValuePatternTest.

File: src/main/java/main/Task.java [hunk 1]
```diff
@@ -1,60 +1,68 @@
...
   public void done() {
     this.firePropertyChange("done", null, null);
   }
-
+  
   private boolean parseFiles() {
     log("Parsing input files.\n");
     tables = new ArrayList<Table>();
-
+
```
Comment (grootstyr) [inline src/main/java/main/Task.java]: This file is full of whitespace errors.

File: src/test/java/export/ExporterTest.java [hunk 2]
```diff
@@ -24,15 +26,16 @@
...
-  Set<String> cols;
+  List<String> cols;
 
   /**
    * Creates a table to give to the exporter.
    * 
    */
   @Before
-  public void setUp() {
+  public void setUp() {
```
Comment (grootstyr) [inline src/test/java/export/ExporterTest.java]: White space

**Comment by grootstyr** [comment]
Formatting is not checkstyle compliant, functionality is good and runs without errors.

**Pull request merged**